src/Command/InfectionCommand.php

```diff
@@ -274,6 +274,11 @@ private function getOutputFormatter(): OutputFormatter
 
     private function applyMemoryLimitFromPhpUnitProcess(Process $process, PhpUnitAdapter $adapter)
     {
+        if (PHP_SAPI == 'phpdbg') {
+            // Under phpdbg we're using a system php.ini, can't add a memory limit there
```

Comment:
maybe we could by still doing a server restart but not disabling xdebug; Might not be easy with Composer XdebugHandler though

Reply:
phpdbg has a default memory limit even on Debian, so this isn't a big concern